ride_service/src/main/java/com/internship/ride_service/util/exceptions/GlobalExceptionHandler.java

```diff
@@ -65,12 +65,12 @@ public ResponseEntity<String> handleHttpMessageNotReadableException(HttpMessageN
                         null,
                         LocaleContextHolder.getLocale()));
     }
-//    @ExceptionHandler({Exception.class})
-//    public ResponseEntity<BaseExceptionDto> handleHttpMessageNotReadableException(Exception ex) {
-//        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
-//                .body(new BaseExceptionDto(messageSource.getMessage(
-//                        UNKNOWN_ERROR.getCode(),
-//                        null,
-//                        LocaleContextHolder.getLocale())));
-//    }
+    @ExceptionHandler({Exception.class})
+    public ResponseEntity<BaseExceptionDto> handleHttpMessageNotReadableException(Exception ex) {
+        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
+                .body(new BaseExceptionDto(messageSource.getMessage(
+                        UNKNOWN_ERROR.getCode(),
+                        null,
+                        LocaleContextHolder.getLocale())));
+    }
```

Comment:
Следует добавить логирование непредвиденного исключения, чтобы можно было определить его причину:
```suggestion
    public ResponseEntity<BaseExceptionDto> handleHttpMessageNotReadableException(Exception ex) {
        log.error("Unexpected exception", ex);
        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
                .body(new BaseExceptionDto(messageSource.getMessage(
                        UNKNOWN_ERROR.getCode(),
                        null,
                        LocaleContextHolder.getLocale())));
    }
```